R/calcualte_age.R

```diff
@@ -21,78 +26,88 @@
 #'
 #' @examples
 #' age <- calculate_age(
-#'   data          = readRDS(system.file("extdata", "test_df.RDS",
-#'                                       package = "cleanepi")),
+#'   data = readRDS(system.file("extdata", "test_df.RDS",
+#'     package = "cleanepi"
+#'   )),
 #'   target_column = "dateOfBirth",
-#'   end_date      = Sys.Date(),
-#'   age_in        = "months",
-#'   na_strings    = "-99"
+#'   end_date = Sys.Date(),
+#'   age_in = "months",
+#'   age_remainder_unit = "days",
```

Comment:
```suggestion
#'   age_remainder_unit = "months",
```

Reply:
Same as above.

Reply:
I will illustrate this in the same example as above.